lib/shared/bucketing/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@devcycle/bucketing",
-  "version": "1.0.21-alpha.0",
+  "version": "1.0.21",
   "license": "MIT",
   "devDependencies": {
-    "@devcycle/bucketing-test-data": "^1.0.12-alpha.0"
+    "@devcycle/bucketing-test-data": "^1.0.11"
```

Comment:
this can stay with the alpha tag for now until we publish the new version of the bucketing sdk